weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewer.tsx

```diff
@@ -161,10 +163,17 @@ export const ObjectViewer = ({
 
     const refValues: RefValues = {};
     for (const [r, v] of _.zip(refs, resolvedRefData)) {
-      if (!r || !v) {
+      if (!r) {
         // Shouldn't be possible
         continue;
       }
+      if (!v) {
+        // Value for ref not found, probably deleted
+        refValues[r] = {
+          _weave_is_deleted_ref: objectRefDisplayName(parseRef(r)).label,
```

Comment:
recommend making some sort of helper like: deletedRefValuePlaceholder() and maybeGetDeleteRefPlaceholder() to co-locate the key naming and structure (as used in `weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewerGroupingCell.tsx`)